core/constants.py

```diff
@@ -85,10 +88,17 @@ def get_package_file_contents(package: str, filepath: str) -> str:
         FileNotFoundError. The file does not exist.
     """
     try:
+        if mode == 'r':
+            with io.open(
+                os.path.join(package, filepath), mode, encoding='utf-8'
+            ) as file:
+                read_mode_data: str = file.read()
+                return read_mode_data
         with io.open(
```

Comment:
On this 97 line I'm getting one lint error which is regarding the encoding. The error is as follows Using open without explicitly specifying an encoding . To read the image file in binary mode I cannot specify the encoding as it raises the error and for this the encoding needs to be None.